test/benchmarks/tests/get_int_string_x10_sqlite3.js

```diff
@@ -1,2 +1,2 @@
 module.exports = require('./get_int_string_x1000_sqlite3').fn.bind(null, 10)
-module.exports.setup = require('./get_int_string_x1000_sqlite3').setupFn.bind(null, 10)
\ No newline at end of file
+module.exports.setup = require('./get_int_string_x1000_sqlite3').setupFn.bind(null, 10)
```

Comment:
As I was working on converting I noticed that the benchmarks for sqlite3 are commented out. Maybe we should clean up these benchmarks completely?